services/core-web/cypress/e2e/majorprojects.cy.ts

```diff
@@ -70,4 +70,57 @@ describe("Major Projects", () => {
       expect(interception.response.statusCode).to.equal(301);
     });
   });
+
+  it("should add a new major project successfully", () => {
+    cy.get('[data-cy="mines-button"]', { timeout: 10000 }).click({
+      force: true,
+    });
+
+    cy.get("#search", { timeout: 10000 }).type("Brenda mine", { force: true });
+
+    cy.get('[data-cy="apply-filter-button"]').click();
+
+    cy.get('[data-cy="mine-link"]', { timeout: 10000 })
+      .eq(0)
+      .click({
+        force: true,
+      });
+
+    cy.get('[data-menu-id^="rc-menu-"][data-menu-id$="permits-and-approvals"]')
+      .scrollIntoView()
+      .trigger("mouseover", { force: true });
+
+    cy.get('[data-cy="major-projects-link"]', { timeout: 3000 }).click({
+      force: true,
+    });
+
+    cy.get('[data-cy="create-new-project"]', { timeout: 3000 }).click({
+      force: true,
+    });
+
+    cy.get("#project_summary_title").type("Cypress Test Project", { force: true });
+    cy.get("#project_summary_description").type("This is just a cypress test project description", {
+      force: true,
+    });
+    cy.get("#mrc_review_required")
+      .contains("No")
+      .click({ force: true });
+    cy.get("#contacts\\[0\\]\\.name").type("Cypress", { force: true });
+    cy.get("#contacts\\[0\\]\\.email").type("cypress@mds.com", { force: true });
+    cy.get("#contacts\\[0\\]\\.phone_number").type("1234567890", { force: true });
+
+    cy.intercept("POST", /.*\/(api\/)?projects\/new\/project-summaries\/new.*$/, {
+      statusCode: 201,
+      body: { message: "project created successfully" }, // Stubbed response
+    }).as("createNewProject");
+
+    cy.get('[data-cy="project-summary-submit-button"]').click({
+      force: true,
+    });
+
+    cy.wait("@createNewProject").then((interception) => {
+      // Assert that the response body contains the expected message
+      expect(interception.response.body.message).to.equal("project created successfully");
+    });
```

Comment:
Could we add in a step here to navigate back to the Permits and Approvals tab and check if the new one shows up in the table?